dwave/cloud/client.py

```diff
@@ -106,10 +106,10 @@ class Client(object):
     Examples:
         This example directly initializes a :class:`~dwave.cloud.client.Client`.
         Direct initialization uses class constructor arguments, the minimum being
-        values for `endpoint` and `token`.
+        a value for `token`.
 
         >>> from dwave.cloud import Client
-        >>> client = Client(endpoint='https://cloud.dwavesys.com/sapi', token='secret')
+        >>> client = Client(token='secret')
```

Comment:
Not sure if this is the right venue for this example, but I would be useful to note that for such "minimal" client, `client.get_solver()` will return "the first" (by id) available online solver. It might happen that's a software solver.

Reply:
Also, to be more specific, user will have to say `client.get_solver(<solver-def-kwargs>)`.

Reply:
Soon this will be covered by "By default, solvers are sorted by average load."